.github/workflows/actions.yml

```diff
@@ -348,7 +348,7 @@ jobs:
           sw_vers
       - name: Get Dependencies
         run: |
-          brew install cmake gettext libdeflate libevent libpsl miniupnpc ninja node pkg-config
```

Comment:
Oh, I see, brew does not build bottles for macOS 11, so it tries to build from source.

Reply:
Yes. So good to approve my fix? :)

Reply:
Honestly, I would like you to try submitting PR to homebrew/core to change default mirror first.
Brew maintainers are very responsive.

Reply:
@fxcoudert  is a Transmission user and is one of the most active homebrew contributors, so it would be good to keep him in the loop when issues like this arise

Reply:
For now, we'll follow:
https://github.com/Homebrew/homebrew-core/pull/167844